docs/concepts/tasks.md

```diff
@@ -53,6 +55,29 @@ Tasks are uniquely identified by a task key, which is a hash composed of the tas
 
     To be clear, there's nothing stopping you from putting all of your code in a single task &mdash; Prefect will happily run it! However, if any line of code fails, the entire task will fail and must be retried from the beginning. This can be avoided by splitting the code into multiple dependent tasks.
 
+**Calling a task's function from another task**
+
+A task can be called from another task by use of the `.fn()` method. Pass any arguments to `fn`. 
```

Comment:
This isn't actually the `fn` method, you're accessing the `fn` attribute which happens to be your function. Perhaps...

```suggestion
Prefect does not allow execution of tasks within other tasks. If you would like to call your task's function directly, you can access it at `task.fn`. You may then call it without any Prefect orchestration or observation.
```